infra/docker/docker-compose.prod.yml

```
@@ -0,0 +1,80 @@
+services:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)